voicer-client/src/components/editor/Editor.jsx

```diff
@@ -1,24 +1,27 @@
-import React from 'react'
-import {HtmlEditor, MenuBar} from '@aeaton/react-prosemirror'
-import {options, menu} from '@aeaton/react-prosemirror-config-default'
+import React, { useState } from "react"
+import { HtmlEditor, MenuBar } from "@aeaton/react-prosemirror"
+import { options, menu } from "@aeaton/react-prosemirror-config-default"
 
-const Editor = ({value, onChange}) =>{
+const Editor = ({ value, onChange }) => {
+  const [mdValue, setMdValue] = useState("")
 
-  return(
+  console.log("value", value)
+  return (
     <HtmlEditor
-    options={options}
-    value={value}
-    onChange={onChange}
-    render={({ editor, view }) => (
-      <div>
-        <MenuBar menu={menu} view={view} />
-        {editor}
-      </div>
-    )}
-  />
+      options={options}
+      value={mdValue}
+      onChange={(e) => {
+        console.log(JSON.stringify(e, null, 2))
```

Comment:
don't forget to get rid of this during prod!

Reply:
@alphaseinor Do you want to get rid of the whole editor or just these four lines?